dashboard/lib/tasks/seed.rake

```diff
@@ -7,21 +7,6 @@ namespace :seed do
     Video.setup
   end
 
-  STANFORD_HINTS_FILE = 'config/stanford-hints-bestPath1.tsv'
```

Comment:
It can be. Though since it has data within it, I intentionally left its removal for a followup PR that will remove all data (drop the DB tables, delete those files, etc.), restricting this PR to removing functionality.